public/tutorials/cross-dom-solidity.js

```
@@ -1,39 +1,65 @@
-await l1Provider.getWithdrawalStatus({ 
-  receipt, 
-  targetChain: l2Provider.chain, 
-}) 
-
-console.log('Relaying message...')
-const [message] = getWithdrawals(receipt)
-await l1Provider.waitToFinalize({ withdrawalHash: message.withdrawalHash, targetChain: l2Provider.chain }) 
-
-console.log('Waiting for message to be relayed...')
-await l1Provider.getWithdrawalStatus({ receipt, targetChain: l2Provider.chain }) 
+  const { createPublicClient, http, createWalletClient } = require("viem");
```

ESLint

[error] Extra semicolon.

(semi)